conda_pack/tests/test_core.py

```diff
@@ -114,6 +109,7 @@ def test_env_properties(py36_env):
     assert 'CondaEnv<' in repr(py36_env)
 
 
+@pytest.mark.skipif(on_win, reason='Activate/deactivate are different on Win')
```

Comment:
Would be good to modify this test to still work on windows, rather than skipping it.

Reply:
doing so now